activity/activity.info

```diff
@@ -6,6 +6,7 @@ bundle_id = org.sugarlabs.AbacusActivity
 exec = sugar-activity AbacusActivity.AbacusActivity
 icon = activity-abacus
 show_launcher = yes
-summary = a tool for simple arithmetic calculations
-category = maths
-repository = https://github.com/sugarlabs/abacus
\ No newline at end of file
+website = https://help.sugarlabs.org/en/abacus.html
```

Comment:
`website` is not defined in the metadata reference.  See https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/bundle/__init__.py and the older https://wiki.sugarlabs.org/go/Development_Team/Almanac/Activity_Bundles#.info_file_format

Before adding `website`, I'd like a patch to `bundle/__init__.py` to add the definition.

Reply:
I was told to reference this https://wiki.sugarlabs.org/go/Development_Team/Almanac/Activity_Bundles#.info_file_format but I was also referring to the active Turtleart activity which had this https://github.com/sugarlabs/turtleart-activity/blob/master/activity/activity.info

Reply:
`Website` could be the placeholder for the help-activity links as we have previously discussed. Hence I suggest we add it to `bundle/__init__.py` and also edit the wiki

Reply:
Thanks.  I'm not surprised you were pointed at only one of the lists of metadata names, but it is only one of the lists; the list in the toolkit is just as important.  Being told is no excuse.  :grin:  Please feed this back to whoever _told_ you.  I'd like to see the Wiki page deprecated in favour of the toolkit documentation.

Yes, I am aware some activities have begun to use `website` before adding it to the lists.  The activities are Turtle Blocks and derivatives, as well as Music Blocks.  I'm sure Walter would be willing to propose the change too, but as you're the one working on this wider issue, you are best placed for this.

While you are at it, you might also;

* make changes for other, as yet "unofficial", names such as `categories`,
* resolve any differences between the Wiki and the toolkit, and;
* deprecate the Wiki page.

Reply:
Reviewed again; waiting for patch to `bundle/__init__.py` in toolkit.